src/main/java/com/sap/oss/phosphor/fosstars/model/value/StringValue.java

```diff
@@ -0,0 +1,69 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import java.util.Objects;
+
+/**
+ * A value for a feature that holds a string.
+ */
+public class StringValue extends AbstractValue<String> {
+
+  /**
+   * Version of an artifact.
+   */
+  private final String content;
+
+  /**
+   * Initializes a value for a feature.
+   *
+   * @param feature The feature.
+   * @param content The versions.
+   */
+  @JsonCreator
+  public StringValue(
+      @JsonProperty("feature") Feature<String> feature,
+      @JsonProperty("content") String content) {
+
+    super(feature);
+    this.content = content;
```

Comment:
```suggestion
    this.content = Objects.requireNotNull(content, "Content can't be null!");
```

Reply:
Do we really not want to allow null values which are interpreted as `isUnknown()`?

Reply:
Yes. You can use `StringValue.unknown()` to create an unknown value.

Reply:
I think we'll need to make `AbstractValue.isUnknown()` final later.